bsm_generator/test/test_bsm_generator_worker.cpp

```diff
@@ -43,6 +43,12 @@ TEST(BSMWorkerTest, testMsgId)
     EXPECT_TRUE(msgId2 != msgId3);
 }
 
+/* added as part of sonarqube bug fix */
+TEST(BSMWorkerTest, testSecMark)
```

Comment:
You might get compile errors because this Test has the same name as the next function. I suggest naming it something else. 
fakeTest instead of testSecMark

Reply:
Okay, I'll go ahead and change that.